lib/puppet/util/yaml.rb

```diff
@@ -9,6 +9,21 @@ module Puppet::Util::Yaml
 
   class YamlLoadError < Puppet::Error; end
 
+  def self.disallowed_class(cls)
```

Comment:
What is the purpose of this ? Is it for catching accidental(?) instances of unsafe classes trying to use YAML safe_load? is it a call back that ruby will invoke when it detects any disallowed classses with safe_load?

Reply:
Remnants from a previous rejected attempt. I pushed a commit that removes this method.